README.md

```diff
@@ -115,9 +115,57 @@ The Cassandra tables in the `tsdb` keyspace (by default) are organized into two
 - **metadata** : tables that enable metadata lookup directly or to enable queries
 - **data** : tables that store the timestamped values
 
-![](docs/tables.drawio.png)
+#### Metadata
 
-To be a bit more explicit, all these tables are written to on ingest.  The "metric_names" table is only read by the "metricNames" metadata query; the "tag_keys" by the "tagKeys" metadata query and the "tag_values" by the "tagValues" metadata query.  The "series_set" and "data_raw" tables are only read by the "query" data query.
+`metric_names`:
+
+&nbsp; | Name | Notes
+---|---|---
+PK | tenant |
+CK | metric_name | 
+&nbsp; | aggregators | `set<text>` of one or more of raw,min,max,sum,count,avg
+
+`tag_keys`:
+
+&nbsp; | Name | Notes
+---|---|---
+PK | tenant |
+PK | metric_name | 
+CK | tag_key |
+
+`tag_values`:
+
+&nbsp; | Name | Notes
+---|---|---
+PK | tenant |
+PK | metric_name | 
+CK | tag_key |
+CK | tag_value |
+
+`series_sets`:
+
+&nbsp; | Name | Notes
+---|---|---
+PK | tenant |
+PK | metric_name | 
+CK | tag_key |
+CK | tag_value |
+CK | series_set | As `{metric_name},{tagK}={tagV},...` with tagK sorted
+
+#### Data
+
+`data_raw`:
+
+&nbsp; | Name | Notes
+---|---|---
+PK | tenant |
+PK | series_set | As `{metric_name},{tagK}={tagV},...` with tagK sorted
+CK | ts | timestamp of ingested metric
+&nbsp; | value | Metric value as double
+
+To be a bit more explicit, all these tables are updated/inserted on ingest.  The "metric_names" table is only read by the "metricNames" metadata query; the "tag_keys" by the "tagKeys" metadata query and the "tag_values" by the "tagValues" metadata query.  The "series_set" and "data_raw" tables are only read by the "query" data query.
```

Comment:
We should maybe remove the last sentence it is probably going to change with time.  If you think it is better to update it, I suggest: 'The "series_set" and "data_raw" tables are read by the "query" data query, and during downsampling.'

Reply:
Good catch. That sentence used to be mostly correct before downsampling :)